src/common/helpers/recordFormatting.helper.ts

```diff
@@ -209,3 +209,15 @@ export const getNonBFMapElemByContainerKey = (
 
   return { key: groupElemKey, value: groupElemValue };
 };
+
+export const applyIntlToTemplates = ({
+  templates,
+  fmt,
```

Comment:
`fmt`? What is it? Please, give variables readable names.

Reply:
`fmt` is a common shorthand for `format`